CONTRIBUTING.md

```diff
@@ -71,15 +71,36 @@ git status
 git commit # type a good commit message
 ```
 
-6. Push your commit to GitHub
+6. Bring master up-to-date and rebase
+
+```sh
+git checkout master
+git pull https://github.com/decred/dcrtime.git

```

Comment:
dcrtime -> dcrdata
But hopefully they already have `master` tracking the remote decred dcrdata repo.  If not, and if it is tracking something else, this could cause a merge and confusion.